.github/workflows/publish-docker-images.yml

```diff
@@ -65,19 +74,19 @@ jobs:
         uses: docker/metadata-action@v4
         with:
           images: getmeili/meilisearch
-          # The latest and `vX.Y` tags are only pushed for the official Meilisearch releases
-          # See https://github.com/docker/metadata-action#latest-tag
+          # Prevent `latest` to be updated for each new tag pushed.
+          # We need latest and `vX.Y` tags to only be pushed for the stable Meilisearch releases.
           flavor: latest=false
           tags: |
             type=ref,event=tag
+            type=raw,value=nightly,enable=${{ github.event_name != 'push' }}
             type=semver,pattern=v{{major}}.{{minor}},enable=${{ steps.check-tag-format.outputs.stable == 'true' }}
-            type=raw,value=latest,enable=${{ steps.check-tag-format.outputs.stable == 'true' }}
+            type=raw,value=latest,enable=${{ steps.check-tag-format.outputs.stable == 'true' && steps.check-tag-format.outputs.latest == 'true' }}
 
       - name: Build and push
         uses: docker/build-push-action@v3
         with:
           # We do not push tags for the cron jobs, this is only for test purposes
-          push: ${{ github.event_name != 'schedule' }}
```

Comment:
We now want to push when we run a cron job, since `nighlty` tag is run at the moment.

Reply:
> since `nightly` tag is run at the moment.
I don't understand this part. Can you explain what you mean?

Reply:
Sorry, not clear indeed,

Before: we ran a cron job every day, that built the docker image on the `main` branch, but did not push any image related to this build. it was only for a check.

Now: we ran a cron job every day, that built the docker image on the `main` branch, but also push this built. The name of the image we push is `nightly`.